store/cachekv/store.go

```diff
@@ -70,6 +78,7 @@ func (store *Store) Get(key []byte) (value []byte) {
 	} else {
 		value = cacheValue.value
 	}
+	store.eventManager.EmitResourceAccessReadEvent("get", key, value)
```

Comment:
For deliver TX we technically only need to add it to the CacheKvStore when validating KVStore objects since we branch to two different CacheKvStores for anteHandler and runMsgs